examples/Linux/README.md

```diff
@@ -0,0 +1,106 @@
+# wolfSentry lwIP Echo Test
+
+This is an extremely basic demo application that uses wolfSentry with the lwIP stack via PCAP on Docker to echo anything sent to it. wolfSentry will filter various types of traffic which can be tested using three Docker nodes the test generates.
+
+## Prerequisites
+
+This test is designed to be run on Linux or macOS, but should work on Windows as well. You need the following installed on your computer:
+
+* Docker - <https://docs.docker.com/get-docker/>
+* docker-compose - <https://docs.docker.com/compose/install/>
+
+## Running echo server
+
+The following command will build the test echo server and start this up along with three testing nodes:
+
+```sh
+sudo docker-compose -f docker-compose.yml up
+```
+
+You can follow the logs for the echo server using:
+
+```sh
+sudo docker-compose -f docker-compose.yml logs -f
+```
+
+It is recommended that you keep the logs following running whilst running the tests.
+
+## Testing
+
+### Accessing nodes
+
+There are three user test nodes to play with. They are named `wolfsentry-test_tester?_1` where `?` is 1, 2 or 3. To log into tester2 as an example:
```

Comment:
As configured, the names are actually `linux_tester?_1`.

Reply:
Doh! Yes, because they take the current directory as the prefix...